lib/classes/Database.js

```diff
@@ -15,31 +15,50 @@ class Database {
 
 	static async initialize() {
 		log("log", "Initializing database...");
-		const configDirectory = await fs.readdir(process.cwd() + "/config");
-		if (!configDirectory.includes("auth.json")) {
-			log("error", "Initialization failed! Authorization file (/config/auth.json/) not found!");
-			process.exit();
-		}
-		const db = require("../../config/auth.json").mariadb;
-		this.sequelizeInstance = new Sequelize(db.database, db.username, db.password, {
-			host: db.host,
-			dialect: "mariadb"
-		});
-		try {
-			await this.sequelizeInstance.authenticate();
+
+		let dbtype = DbConfig.dbtype;
+		this.sequelizeInstance = this.#dbInstance[dbtype]();
```

Comment:
Needs optional chaining to prevent `TypeError` throwing on unexpected `dbType`. Also, create fallback to some default `dbType` value or just send log message about incorrect database type and stop execution.